aiida_wannier90/calculations.py

```diff
@@ -257,10 +242,27 @@ def prepare_for_submission(self, folder):  #pylint: disable=too-many-locals, too
         if pp_setup:
             param_dict.update({'postproc_setup': True})
 
-        if 'local_input_folder' not in self.inputs and 'remote_input_folder' not in self.inputs and not pp_setup:
-            raise exc.InputValidationError(
-                'Either local_input_folder or remote_input_folder must be set.'
-            )
+        has_local_input = 'local_input_folder' in self.inputs
+        has_remote_input = 'remote_input_folder' in self.inputs
+        if pp_setup:
+            if has_local_input or has_local_input:
+                raise exc.InputValidationError(
+                    "Can not set 'local_input_folder' or 'remote_input_folder' "
+                    "with the 'postproc_setup' option."
+                )
+
+        else:
+            if has_local_input and has_remote_input:
+                raise exc.InputValidationError(
+                    "None of the 'local_input_folder' and 'remote_input_folder' "
+                    "inputs is set. Exactly one of the two must be given."
+                )
+            if not (has_local_input or has_remote_input):
+                raise exc.InputValidationError(
+                    "Both the 'local_input_folder' and 'remote_input_folder' "
+                    "inputs are set, but they are exclusive. Exactly one of "
+                    "the two must be given."
+                )
```

Comment:
I think the two error messages are swapped

Reply:
Fixed.